recipes/folly/config.yml

```diff
@@ -1,7 +1,9 @@
 versions:
-    "2019.10.21.00":
-      folder: all
-    "2020.08.10.00":
-      folder: all
-    "2022.01.31.00":
-      folder: all
+  "2019.10.21.00":
```

Comment:
Maybe it's worth while dropping some of the older versions?

Reply:
Removed 2019.10.21.00, I would like to keep 2020 version for now